velox/exec/HashBuild.cpp

```diff
@@ -735,10 +735,14 @@ bool HashBuild::finishHashBuild() {
         }
       }
 
-      const bool hasOthers = !otherTables.empty();
+      // TODO: we need to disable parallel join build before
```

Comment:
nit: "TODO: Re-enable after xxx is fixed" since to-do is to re-enable, not to disable :-)